plugins/reolink/src/main.ts

```diff
@@ -50,9 +52,9 @@ class ReolinkCamera extends RtspSmartCamera implements Camera, Reboot, Intercom,
                 this.updatePtzCaps();
             },
         },
-        doorbellUseOnvifDetections: {
-            hide: true,
-            defaultValue: true,
+        useOnvifDetections: {
```

Comment:
Move this under the `group` Advanced or something where it doesn't encourage a user to change it. Should use intelligent defaults typically.

Reply:
Got it.  Enabled if doorbell is auto-detected.

Reply:
Sorry should have been more clear, make have a property  "choices" entry with:

"Default",
"Enabled",
"Disabled"

Reply:
Reolink provides a list of ONVIF cameras here: https://support.reolink.com/hc/en-us/articles/900000617826/

Based on a sample size of 3 (Reolink Video Doorbell PoE, CX410, Reolink TrackMix PoE [from the existing code]) then deviceInfo.model looks like the field we can use for a smarter default, however the list will need to be maintained going forward.

Reply:
Alright, updated to use the list.

Reply:
> How about "Enabled" and "Disabled" ? At the moment, a default value is hard to determine since I'd need to create a white list of all cameras. At the moment, I only know for sure that the doorbell and CX410 support ONVIF object detection.

Sorry I should have provided clarification: The default behavior, prior to these changes, is to use ONVIF if the device is a doorbell. That has a verified fully functional ONVIF implementation. That is the behavior to preserve.
I don't think using the list there will be exhaustive/future proof. Furthermore, I think some of these cameras don't report object or motion detection events over ONVIF.

So the change should be: Default/Enabled/Disabled.

Default value:
Doorbell: Enabled
Not Doorbell: Disabled. We can expand this support with manually verified camera models.

Reply:
Ok I see.  That makes the list a lot shorter then.  Updated.